ui/apps/platform/src/Components/ReportJob/PartialReportModal.tsx

```diff
@@ -63,6 +79,22 @@ function PartialReportModal({ failedClusters = [], onDownload }: PartialReportMo
                         cluster logs to diagnose the issue. The following clusters are not included
                         in this report
                     </FlexItem>
+                    <Toolbar className="pf-v5-u-w-100">
+                        <ToolbarContent>
+                            <ToolbarItem variant="pagination" align={{ default: 'alignRight' }}>
+                                <Pagination
+                                    isCompact
+                                    itemCount={failedClusters.length}
+                                    page={page}
+                                    perPage={perPage}
+                                    onSetPage={(_, newPage) => setPage(newPage)}
+                                    onPerPageSelect={(_, newPerPage) => {
+                                        setPerPage(newPerPage);
```

Comment:
Nit for consistency with behavior elsewhere:
```suggestion
                                        setPerPage(newPerPage);
                                        setPage(1);
```